### PR TITLE
deletes unused scheduled event model and its dependencies

### PR DESCRIPTION
**Pull request by saikumar9**
# PR Checklist

Please check if your PR fulfills the following requirements
- [x] The title follows our [guidelines](https://github.com/ideacrew/enroll/blob/trunk/CONTRIBUTING.md#commit
)
- [x] Tests for the changes have been added (for bugfixes/features)

# PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature (requires Feature flag)
- [ ] Data fix or migration (inert code, no impact until run)
- [x] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Dependency updates (e.g., add a new gem or update to a version)

# What is the ticket # detailing the issue?

Ticket: [IVL Product Development - 187177636](https://www.pivotaltracker.com/story/show/187177636)

# A brief description of the changes

Current behavior: All the endpoints of the controller `Exchanges::ScheduledEventsController` do not have authorization.

New behavior: The controller, models, views(HTML and JavaScript), Rspecs, and cucumbers are deleted as the feature/code is not being used. This feature/code is considered legacy/dead. The code was implemented 7 years ago in the year 2017.

# Feature Flag

For all new feature development, a feature flag is required to control the exposure of the feature to our end users. A feature flag needs a corresponding environment variable to initialize the state of the flag. Please share the name of the environment variable below that would enable/disable the feature and which client(s) it applies to.

Variable name:

- [ ] DC
- [ ] ME

# Additional Context
Include any additional context that may be relevant to the peer review process.

# AppScan CodeSweep Failure
In the event of a failed check on the AppScan CodeSweep step of our GitHub Actions workflow, please review the False Positive protocol outlined here: appscan_codesweep/CODESWEEP_FALSE_POSITIVES_README.MD

Add all required notes to this section if the failure is a suspected false positive.

### CODE REVIEW

**Review by polographer — DISMISSED** [review]

File: app/helpers/config/aca_helper.rb [hunk 1]
```diff
@@ -290,10 +290,6 @@ def site_broker_claim_quoting_enabled?
       Settings.site.broker_claim_quoting_enabled
     end
 
-    def calendar_is_enabled?
```
Comment (raghuramg) [inline app/helpers/config/aca_helper.rb:293]: Can we remove calendar_enabled setting as well...

`./config/client_config/me/config/settings.yml:96:  calendar_enabled: true
./config/client_config/dc/config/settings.yml:93:  calendar_enabled: true
./config/settings.yml:73:  calendar_enabled: true
./components/notifier/config/settings.yml:93:  calendar_enabled: true
./components/notifier/spec/dummy/config/settings.yml:96:  calendar_enabled: true
./components/transport_profiles/config/settings.yml:93:  calendar_enabled: true
./components/transport_profiles/spec/dummy/config/settings.yml:95:  calendar_enabled: true
./components/ui_helpers/config/settings.yml:93:  calendar_enabled: true
./components/ui_helpers/spec/dummy/config/settings.yml:96:  calendar_enabled: true
./components/transport_gateway/config/settings.yml:93:  calendar_enabled: true
./components/transport_gateway/spec/dummy/config/settings.yml:96:  calendar_enabled: true
./components/benefit_markets/config/settings.yml:93:  calendar_enabled: true
./components/benefit_markets/spec/dummy/config/settings.yml:96:  calendar_enabled: true
./components/benefit_sponsors/config/settings.yml:93:  calendar_enabled: true
./components/benefit_sponsors/spec/dummy/config/settings.yml:96:  calendar_enabled: true
./components/financial_assistance/config/settings.yml:93:  calendar_enabled: true
./components/sponsored_benefits/config/settings.yml:93:  calendar_enabled: true
./components/sponsored_benefits/spec/dummy/config/settings.yml:96:  calendar_enabled: true`

**Review by raghuramg — CHANGES_REQUESTED** [review]

File: app/helpers/config/aca_helper.rb (hunk 1 above)
Comment (saikumar9) [reply app/helpers/config/aca_helper.rb:293]: I did not know that we have a legacy setting. Good Catch @raghuramg. I will work on it.

**Review by raghuramg — APPROVED** [review]
removed all the references for schedule events.

code changes looks good to me

**Review by nisanthyaganti9 — APPROVED** [review]
LGTM

**Pull request merged**